<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>479</width>
    <height>103</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>479</width>
    <height>102</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>用户引导上色</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>481</width>
     <height>51</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="imgTab">
    <attribute name="title">
     <string>待上色图像</string>
    </attribute>
    <widget class="QLabel" name="imgLabel">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="sketchTab">
    <attribute name="title">
     <string>引导图像</string>
    </attribute>
    <widget class="QLabel" name="sketchLabel">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QFrame" name="buttonFrame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>481</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="leftMargin">
     <number>15</number>
    </property>
    <property name="topMargin">
     <number>11</number>
    </property>
    <property name="rightMargin">
     <number>15</number>
    </property>
    <property name="bottomMargin">
     <number>11</number>
    </property>
    <item>
     <widget class="QPushButton" name="btnLoadImg">
      <property name="text">
       <string>选取图像</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnLoadSketch">
      <property name="text">
       <string>选取引导图</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnColorize">
      <property name="text">
       <string>上色</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnSaveImg">
      <property name="text">
       <string>保存图片</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
